services/azure-servicebus/src/main/java/com/microsoft/windowsazure/services/servicebus/models/GetEventHubResult.java

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
What is the purpose of this class? Seems to simply hold an instance of `EventHubInfo` and doesn't convey much information. Not to mention that a result should not be mutable.